### DISCUSSION TITLE
Mahsaban.com مهسا بان

### DISCUSSION THREAD

**Discussion by bidar-ca** [WeAreMahsaAmini/Core#209] [Ideas]
Mahsaban.com
ما همه مهسا هستیم

هدف ساخت اپلیکیشن موبایل مهسابان برای اعلام درخواست کمک مخصوص زنان و دختران در مواقع تهدید به آزار

نرم افزار شبکه اجتماعی که افراد حامی (داوطلبین) در هر شغلی که هستند بتوانند از اعلام نیاز با خبر شوند و به کمک فرد (مورد تهدید ) بیایند

مثال:
دختری در خیابان با هر نوع پوششی احساس خطر می کند
روی این اپ موبایل کد احساس تهدید را می زند
افراد داوطلب در نزدیکی محل تهدید مطلع می شوند

عنوان پیشنهادی برای افراد داوطلب که پاس داری از حریم بانوان را تقبل می کنند
نام های «سپاه مهسا» یا «مهسا بان» یا «ژینا بان»

این ایده برای جلوگیری هر گونه مواجهه مزاحمت فیزیکی است
متعهد داشتن افراد به حفظ حریم شخصی،
تقویت حس امنیت فیزیکی از دختران و خانم ها در هر ساعتی از شبانه روز

 

Instant help request

mahsaban.com

https://t.me/mahsaban

**Comment by bidar-ca** [comment]
سپاس از همه دوستانی که به این پیشنهاد رای دادند 

لطفا سایت mahsaban.com را ببینید و پیشنهاد خودتان را بدهید .

**Comment by ksaffarian** [comment]
من فکر میکنم اساسا حفاظت از جان و حقوق انسانها، یکی از وظایف اولیه حاکمیت است که در مقابل شهروندان و مردم بر عهده دارد. (البته قبول دارم که این وظیفه در حال حاضر بخوبی اجرا نمیشود).
بنابراین ورود به این موضوع ممکن است مشکلات حقوقی در پی داشته و کار پیچیده ایست.
لذا، شاید در این راستا، بتوانیم پلتفرمی برای اطلاع رسانی فرهنگی درست کنیم که از یک طرف روحیه حمایت گری مردم افرایش یافته و طرفی دیگر مطالبه گری متمرکز از ارگانهای انتظامی و یا نیروهای محافظتی داشت.

**Reply by bidar-ca** [reply]
بله همین طور است فرمودید

ولی بعضی مواقع ( در همه جای دنیا ) ممکن است این موقعیت بوجود بیاید که شخصی احساس نا امنی (یا احساس خطر) کند 

در این زمان یکی از دکمه های:
1- "اطلاع به خانواده ( مثل پدر و مادر) " و یا
2- "دکمه اطلاع به دوستان نزدیک" و یا 
3- دکمه "اطلاع به حامیان نزدیک در محل" 
را بزند و آنها را از موقعیت خود مطلع کند.

**Reply by ksaffarian** [reply]
ایده "اطلاع رسانی از موقعیت" را دوست داشتم و فکر میکنم لازم هست.
البته میدونیم که این موضوع، در ایران مثل چاقوی دو لبه عمل میکنه
در همه جای دنیا، نیروهای سرکوبگر و لباس شخصی مثل ایران نیستند.
این پلیکیشن میتونه توسط مثلا بسیجی ها برای‌ کاربردهای‌مشابه هم استفاده بشه.
در نهایت، ایده کاربردی هست.
البته از این موضوع غافل نشیم که قابلیت sos روی گوشی ها هست، اما اون دیگه برای موارد خیلی اضطرار مصرف میشه.

**Reply by arshamalh** [reply]
برداشتی که من از این ایده داشتم،
محافظت از دختران، در مقابل حکومت و گشت ارشاد بود،
بنابراین مطالبه گری از ارگان هایی که خودشون عامل های اصلی مزاحمت هستن کاملا بی معنیه.
و اگه بخواید با خطکش ج.ا بسنجید، کل این پروژه مشکلات حقوقی در پی داره.
ولی دم سازنده هاش و کانتریبوتر هاش گرم که با وجود مشکلات ممکن، به کارشون ادامه میدن.

**Reply by bidar-ca** [reply]
هدف آرامش بخشی به دختران است 

حفاظت و مراقبت از آزادی و شادی همه دختران و زنان با پوشش آزاد 

با اولویت خودش ( اطلاع به خانواده، دوستان نزدیکش، و یا افراد مورد تاییدی که در اطرافش هستند )

**Comment by IJustWantSomeFreedom** [comment]
همچین ایده ای با فریاد زدن دختر برای کمک تفاوت چندانی نداره. فرد اگر خطری احساس کنه، اینکه داد بزنه کمک بخواد و اینکه سریعا محل رو ترک کنه به نظر من خیلی سریعتر و راحتتر از اینکه اپی رو باز کنه، دکمه رو بزنه و منتظر بمونه که یکی کمکش کنه (اون هم در شرایطی که اینترنت و vpn داشته باشه) و همه افراد نزدیک اون محل، این اپ رو داشته باشن.

ولی میشه ایده sos رو که اتفاقا یکی از دوستان مطرح کردن بودن به صورت advanced تر پیاده کنیم.
که یه اپی باشه در صورت خطر دکمش رو بزنیم و لوکیشن، ویدیو چند ثانیه ای، پیام از قبل آماده شده و ... ارسال بشه به خونواده طرف. همچنین لوکیشن لایو هم میشه فرستاد تا خونواده تا آخرین لحظه از محل فرد مطلع باشن و هر زمانی که خواستن با زدن دکمه، به دوربین و میکروفون گوشی دسترسی داشته باشن.

**Comment by nadimontreal** [comment]
بله ، اطلاع رسانی به خانواده و افراد مورد اعتماد خیلی مهم است

**Comment by nadimontreal** [comment]
mahsaban.com

**Comment by nadimontreal** [comment]
.
